.storybook/constants.ts

```diff
@@ -31,4 +31,7 @@ const nodeVersionData: NodeVersionData[] = [
   },
 ];
 
+// TODO: Hardcode some values
+const nodeReleasesData: NodeReleaseData[] = [];
```

Comment:
we should create an file like [that](https://github.com/nodejs/nodejs.org/blob/f0c376bf4404ca2751b24a96681f6ed5edf65b37/__fixtures__/page.tsx)

Reply:
Yes, I will, thanks!

However, I am currently looking to get inputs regarding the overall solution first. 😄 
Let's ignore the TODOs as I will get to them later.

Reply:
@HinataKah0 lmkw once you made the fixtures as that'd be unblocking another PR!

Reply:
Done, please take a look!

Reply:
For me fixture is perfect